posts/epiparameter_v0.4.0/index.qmd

```diff
@@ -0,0 +1,39 @@
+---
+title: "epiparameter v0.4.0"
+author:
+  - name: "The Epiverse-TRACE development team"
+date: "2025-01-08"
+categories: [new-release]
+---
+
+We are very excited to announce the release of a new [https://epiverse-trace.github.io/epiparameter](epiparameter) version v0.4.0.
```

Comment:
This is printing the link since f276a91aa5e420e9f88ef53811cdf49f8181c485. Do we not want it to be the name of the package hyperlinked?

Reply:
I've updated this in f5c17a2.

Reply:
If the release template as it is currently on `main` is incorrect, I've made a fix for this in PR #357.